code/__DEFINES/dcs/signals/signals_food.dm

```diff
@@ -13,12 +13,6 @@
 /// called when an edible ingredient is added: (datum/component/edible/ingredient)
 #define COMSIG_FOOD_INGREDIENT_ADDED "edible_ingredient_added"
 
-// Deep frying foods
-/// From obj/item/food/deepfryholder/Initialize
-#define COMSIG_ITEM_FRIED "item_fried"
```

Comment:
Can you keep the signal, even if you have no consumers? It is the best way to implement #71550.